release_bot/github.py

```diff
@@ -172,6 +172,9 @@ def add_comment(self, subject_id):
         """Add self.comment to subject_id issue/PR"""
         if not subject_id or not self.comment:
             return
+        if self.conf.dry_run:
+            self.logger.info("I would add a comment to the pull request created.")
+            return "SKIPPED"
```

Comment:
the expectation of response in this method is to get HttpResponse object, you are returning a string here, that's not good

Reply:
Feel free to just return None